Makefile

```diff
@@ -28,6 +28,16 @@ VERSION_STRING=$(shell /usr/libexec/PlistBuddy -c "Print :CFBundleShortVersionSt
 all: bootstrap
 	$(BUILD_TOOL) $(XCODEFLAGS) build
 
+sourcery:
+	sourcery --sources Tests --templates .sourcery/LinuxMain.stencil --output .sourcery
```

Comment:
If you add to the top of the template the target file, it will override it instead of creating .generated.swift 
`// sourcery:file:Tests/LinuxMain.swift` 
https://twitter.com/ilyapuchka/status/847567291302674434
https://oleb.net/blog/2017/03/keeping-xctest-in-sync/